cmd/internal/server/core.go

```diff
@@ -148,6 +172,10 @@ func (s *Server) stopFUSEServer() {
 }
 
 func (s *Server) shutdown() {
+	if s.forVerifyInstall {
+		return
+	}
+
 	if s.opts.CPUProfilePath != "" {
 		pprof.StopCPUProfile()
 	}
```

Comment:
Wouldn't hurt to still make sure we flush open journals.

Reply:
Deferring that to when we need it since nothing should be printed to journals during `verify-install`